openquake/commands/plot.py

```diff
@@ -710,6 +721,373 @@ def make_figure_tot_curves(extractors, what):
+        afe_RTGM.append(_find_afe_target(
+            imls[m], AFE[m], rtgm_probmce[m]))
+
+    plt = import_plt()
+    plt.figure(figsize=(12, 9))
+    plt.rcParams.update({'font.size': 16})
+    colors = mpl.colormaps['viridis'].reversed()._resample(3)
+    patterns = ['-', '--', ':']
+    for i, imt in enumerate(imts):
+        lab = _get_label(imt)
+        plt.loglog(imls[i], AFE[i], color=colors(i), linestyle=patterns[i],
+                   label=lab, linewidth=3, zorder=1)
+        # plot the label only once but it must be at the end of the legend
+        if imt == imts[-1]:
+            plt.loglog([rtgm_probmce[i]], [afe_RTGM[i]], 'ko',
+                       label='Probabilistic MCE',  linewidth=2,
+                       markersize=10, zorder=3)
+        else:
+            plt.loglog([rtgm_probmce[i]], [afe_RTGM[i]], 'ko',
+                       linewidth=2, markersize=10, zorder=3)
+        plt.loglog([numpy.min(imls[i]), rtgm_probmce[i]],
+                   [afe_RTGM[i], afe_RTGM[i]],
+                   'darkgray', linestyle='--', linewidth=1)
+        plt.loglog([rtgm_probmce[i], rtgm_probmce[i]],
+                   [0, afe_RTGM[i]], 'darkgray',
+                   linestyle='--', linewidth=1)
+    plt.grid('both')
+    plt.legend(fontsize=13)
+    plt.xlabel('Acceleration (g)', fontsize=20)
+    plt.ylabel('Annual frequency of exceedance', fontsize=20)
+    plt.legend(loc="best", fontsize='16')
+    plt.ylim([10E-6, 1.1])
+    plt.xlim([0.01, 4])
+    if update_dstore:
+        if Image is not None:
+            bio = io.BytesIO()
+            plt.savefig(bio, format='png', bbox_inches='tight')
+            logging.info('Storing png/hcurves.png')
+            dstore['png/hcurves.png'] = Image.open(bio)
+        else:
+            logging.warning(
+                'Missing module PIL: skipping storing png/hcurves.png')
+    return plt
+
+
+def plot_governing_mce(dstore, update_dstore=False):
+    """
+    :param dstore: the datastore
+    :returns: image of governing MCE
+    """
+    dinfo = get_info(dstore)
+    # get imls and imts, make arrays
+    imtls = dinfo['imtls']
+    plt = import_plt()
+    js = dstore['asce7'][()].decode('utf8')
+    dic = json.loads(js)
+    MCEr = [dic['PGA'], dic['SS'], dic['S1']]
+    T = [from_string(imt).period for imt in imtls]
+
+    limit_det = [0.5, 1.5, 0.6]
+    # presenting as maximum component -> do not need conversion facts
+    RTGM = dstore.read_df('rtgm')['ProbMCE']
+    plt.figure(figsize=(8, 6))
+    plt.rcParams.update({'font.size': 15})
+    plt.plot(T, limit_det, 'kx', markersize=15, label='DLL', linewidth=1)
+    plt.plot(T[0], RTGM[0], 'bX', markersize=12, label='$PGA_{GM}$',
+             linewidth=3)
+    plt.plot(T[1:], RTGM[1:], 'bs', markersize=12,
+             label='$S_{S,RT}$ and $S_{1,RT}$', linewidth=3)
+    MCEr_det = [dic['PGA_84th'], dic['SS_84th'], dic['S1_84th']]
+    if any([val == 'n.a.' for val in MCEr_det]):  # hazard is lower than DLLs
```

Comment:
I am not sure this assumption is always correct

Reply:
It is always correct, no probkem